apps/server/src/modules/task/service/submission.service.ts

```diff
@@ -26,4 +31,104 @@ export class SubmissionService {
 
 		await this.submissionRepo.delete(submission);
 	}
+
+	async deleteSubmissionsByUserId(userId: EntityId): Promise<DomainOperation> {
+		this.logger.info(
+			new DataDeletionDomainOperationLoggable(
+				'Deleting data from Submissions',
```

Comment:
Please change it to some more meaningful and descriptive value, e.g., `Deleting Submissions owned by user`.